packages/cli/package.json

```diff
@@ -80,6 +82,7 @@
     "lodash": "^4.17.15",
     "multiaddr": "^10.0.1",
     "peer-id": "^0.15.3",
+    "prom-client": "^14.0.1",
```

Comment:
Why is `"prom-client"` a dependency of the CLI?

Reply:
not sure?? will delete

Reply:
Oh, its there for the `Registry` type, used in the beacon handler